workshops/views.py

```diff
@@ -178,6 +178,16 @@ class SiteUpdate(LoginRequiredMixin, UpdateViewContext):
     slug_url_kwarg = 'site_domain'
     template_name = 'workshops/generic_form.html'
 
+
+@login_required
+def site_delete(request, site_domain):
+    """Delete specific site."""
+    site = get_object_or_404(Site, domain=site_domain)
+    site.delete()
+
+    messages.success(request, 'Site was deleted successfully.')
```

Comment:
Hmm.  I hadn't noticed out [`django.contrib.messages`][1] use before (looks like it came in with b05d4bd2 (First pass, not processed yet, 2014-12-22, #131).  But “[The messages framework allows you to temporarily store messages in one request and retrieve them for display in a subsequent request (usually the next one)][1]” sounds like a lot of state to store in the expectation that you're going to get a particular future request.  Do we really need this here?

[1]: https://docs.djangoproject.com/en/1.8/ref/contrib/messages/